lib/vtls/openssl.c

```diff
@@ -384,6 +384,31 @@ static char *ossl_strerror(unsigned long error, char *buf, size_t size)
   return buf;
 }
 
+/* Return an extra data index for the connection data.
+ * This index can be used with SSL_get_ex_data() and SSL_set_ex_data().
+ */
+static int ossl_get_ssl_conn_index(void)
+{
+  static int ssl_ex_data_conn_index = -1;
```

Comment:
Will the use of `static` like this not make this function (possibly) fail when used threaded ?

Reply:
Aha, thanks!